app/assets/stylesheets/accounts.scss

```diff
@@ -34,6 +34,7 @@
     text-align: center;
     position: relative;
     z-index: 2;
+    text-shadow: 0 0 2px #000;
```

Comment:
There are other places where full black color is used, and it is using that variable. For me it's about consistency in the code. I don't expect anyone to actually change color8's value.